### ISSUE TITLE
DateTime fields are returned in local time rather than UTC

### ISSUE THREAD

**Issue by philjt** [linvi/tweetinvi#305]
Looking at the API docs for twitter it states that the datetime values are returned in UTC, therefore I was expecting Tweetinvi to return these values in UTC, however they are returned in Local time.

Is this by design or unexpected? I can not see anything in the documentation to state either way.

Example:
Twitter Date
Tue Jul 19 09:24:53 +0000 2016

From DateTime.ParseExact:
{19/07/2016 10:24:53}

THIS HAS CONVERTED TO LOCAL TIME (BST)!

Thanks

**Comment by haseeb1431** [comment]
There isn't any conversion being done in API specifically. 
Can you show your code what you're trying? I feel like you're using DateTime parse method properly which is converting it to your time zone.

**Comment by philjt** [comment]
I was asking what is the expected behaviour of the datetime properties in
tweetinvi, such as the tweet.created_at property. I had set my expectations
based on the Twitter API docs that stated these properties were returned in
UTC.

However when I used these values returned by tweetinvi they are being
converted to local time, this I wasn't expecting.

I have looked at the code for tweetinvi and seen you are using
ParesDateExact and can see this is returning a local time.

So my question really is this behaviour by design? Are you wanting to
return the datetime as local time? If so then I need to change my code to
convert back to UTC as that is the format I need.

Thanks
Phil
